### DISCUSSION TITLE
Obtainig parent domain of the current url

### DISCUSSION THREAD

**Discussion by koiuo** [qutebrowser/qutebrowser#5624] [Q&A] [answered]
Hi, what's the easiest way to implement something like

```
tSH: config-cycle -p -u *://*.{url:host}/* content.javascript.enabled ;; reload
```

but so that parent domain of the url is used.

For example if the url is `https://accounts.google.com/`, the command would enable javascript for `*://*.google.com/*`.

**Comment by The-Compiler** [comment, marked as answer]
I assume for something like `example.com` you'd still want `example.com` and not `*.com`, right? And for `example.accounts.google.com` you'd probably still want `accounts.google.com`? If so, you can write a [userscript](https://github.com/qutebrowser/qutebrowser/blob/master/doc/userscripts.asciidoc) using something like [tldextract](https://github.com/john-kurkowski/tldextract) to do this. There's an example here: https://github.com/qutebrowser/qutebrowser/pull/5460#issuecomment-642319563
